scapy/route6.py

```diff
@@ -288,12 +317,12 @@ def route(self, dst=None, dev=None, verbose=conf.verb):
         best_plen = (paths[0][0], paths[0][1])
         paths = [x for x in paths if (x[0], x[1]) == best_plen]
 
-        res = []
-        for p in paths:  # Here we select best source address for every route
-            tmp = p[2]
-            srcaddr = get_source_addr_from_candidate_set(dst, tmp[1])
+        res = []  # type: List[Tuple[int, int, Tuple[str, str, str]]]
+        for path in paths:  # we select best source address for every route
+            tup = path[2]
```

Comment:
tmp is better than tup IMHO.

Reply:
For sure but `tmp` is used a bit later with a different type :/

Reply:
I understand. tmp_candidates or candidates will be perfect.

Reply:
I chose `tmp_c` so that it doesn't exceed the max line lengthS